internal/cmd/db_create.go

```diff
@@ -137,15 +142,15 @@ func getDatabaseName(args []string) (string, error) {
 }
 
 // Returns (group, error)
-func groupFromFlag(client *turso.Client) (turso.Group, error) {
-	groups, err := getGroups(client)
-	if err != nil {
-		return turso.Group{}, err
-	}
+func groupFromFlag(groups []turso.Group) (turso.Group, error) {
 
 	if groupFlag != "" {
 		if !groupExists(groups, groupFlag) {
-			return turso.Group{}, fmt.Errorf("group %s does not exist", groupFlag)
+			groupNames := make([]string, len(groups))
+			for i, g := range groups {
+				groupNames[i] = g.Name
+			}
+			return turso.Group{}, fmt.Errorf("group %s does not exist. You have the following groups: %v. You can use 'turso group create' to create a new group", groupFlag, strings.Join(groupNames, ", "))
```

Comment:
List can be pretty large here, no?

Reply:
Are you suggesting it can be so large that we should truncate it?

Reply:
I think we can keep previous logic, no? Where we didn't show the list in the error message